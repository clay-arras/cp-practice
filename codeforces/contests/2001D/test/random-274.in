1
19
7 5 18 5 25 7 13 9 8 18 19 22 16 10 13 9 18 22 14
