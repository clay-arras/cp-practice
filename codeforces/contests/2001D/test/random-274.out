12
18 5 25 7 13 8 19 16 10 9 22 14
